lib/super_digit.rb

```diff
@@ -1,15 +1,25 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n)
+# Space Complexity - O(n)
 def super_digit(n)
-   
-end
-  
+  return n if n <= 9
+  return super_digit(n.digits.sum) if n > 9
```

Comment:
You don't need the if since line 6 will return if it's not true.

```suggestion
  return super_digit(n.digits.sum) 
```